frontend/dispatcher.lua

```diff
@@ -967,7 +967,7 @@ function Dispatcher:_showAsMenu(settings)
     end
     local ButtonDialogTitle = require("ui/widget/buttondialogtitle")
     quickmenu = ButtonDialogTitle:new{
-        title = settings.settings.name or "Quick Menu",
+        title = settings.settings.name or _("QuickMenu"),
```

Comment:
Is the space removal on purpose?

Reply:
Given our usual style I think it should probably be changed as follows:
```suggestion
        title = settings.settings.name or _("Quick menu"),
```